pkgs/development/python-modules/video2midi/default.nix

```diff
@@ -1,24 +1,24 @@
-{ stdenv, fetchFromGitHub, python2Packages, opencv3 }:
+{ stdenv, fetchFromGitHub, pythonPackages, opencv3 }:
 
 let
-  opencv3_ = python2Packages.toPythonModule (opencv3.override {
+  opencv3_ = pythonPackages.toPythonModule (opencv3.override {
```

Comment:
pythonPackages, python2Packages, and python27Packages are all aliases of each other.

Reply:
@jonringer Ok. But since it now supports both python 2 and 3, I don't want to show "2" here to avoid misunderstandings

Reply:
Try doing `python3Packages`, this will use python3.7 and related packages.

I'm all for ripping out python2, please do so :)